packages/lumx-react/src/components/avatar/Avatar.tsx

```diff
@@ -47,6 +51,9 @@ const CLASSNAME = getRootClassName(COMPONENT_NAME);
 const DEFAULT_PROPS: Partial<AvatarProps> = {
     size: Size.m,
     theme: Theme.light,
+    thumbnailProps: {
```

Comment:
Shouldn't we add the AspectRatio prop on the thumbnailProps omit list as we want to force square aspect ratio (avatar is always a circle)?